README.md

```diff
@@ -56,6 +58,16 @@ Once it's activated, the extension will automatically trigger itself whenever yo
 
 ![screenshot](https://user-images.githubusercontent.com/38117856/107463541-542e8500-6b2c-11eb-8b25-082f344c1587.png)
 
+### Bookmarklet
+
+The online tool can be used as a bookmarklet. To add one manually, press Ctrl+D to create a new bookmark and add this code in the URL field:
+
+```js
+javascript:!function(){if(m=window.location.href.match(/github.com\/([^\/]+)\/([^\/]+)/),m){window.open(`https://useful-forks.github.io/?repo=${m[1]}/${m[2]}`)}else window.alert("Not a GitHub repo")}();
+```
+
+When you're viewing a GitHub repo in your browser, you can press the newly created bookmarklet and this will open the online tool for the repo.
```

Comment:
```suggestion
When you're viewing a GitHub repo in your browser, you can press the newly created bookmarklet and this will open the online tool for the repo in a new tab.
```